site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -164,6 +164,22 @@ include::_tenant-generator-type-config-rules.adoc[]
 [field]#tenant.externalIdentifierConfiguration.emailVerificationIdTimeToLiveInSeconds# [type]#[Integer]# [required]#Required# [since]#Available since 1.8.0#::
 The time in seconds until a email verification Id is no longer valid and cannot be used by the Verify Email API. Value must be greater than 0.
 
+[field]#tenant.externalIdentifierConfiguration.emailVerificationOneTimeCodeGenerator.length# [type]#[Integer]# [optional]#Optional# [default]#defaults to 6# [since]#Available since 1.27.0#::
++
+:generator_type: emailVerificationIdGenerator
+:generator_display_name: email verification one time code
+include::_tenant-generator-length-config-rules.adoc[]
+:generator_type!:
+:generator_display_name!:
++
+
+[field]#tenant.externalIdentifierConfiguration.emailVerificationOneTimeCodeGenerator.type# [type]#[String]# [required]#Required# [default]#randomAlphaNumeric# [since]#Available since 1.27.0#::
```

Comment:
Optional, and has a default.  

https://github.com/FusionAuth/fusionauth-java-client/blob/39ed46a3380816695d987988f676dc37df51abfe/src/main/java/io/fusionauth/domain/ExternalIdentifierConfiguration.java#L26